coralproject/talk
# Added translation to staff label in comment stream.

## What does this PR do?

Apply translation to the 'Staff' label that appears in comment stream after the username.
The same translation is applied that's also used in admin interface when assigning a role: ``community.staff``

## How do I test this PR?

- Setup translation for ``community.staff``
- Add comment or reply to a discussion
- Notice the label no longer always being 'Staff', but show the translation that was added.
